### ISSUE TITLE
Enable community-wide participation in large-scale curriculum changes

### ISSUE THREAD

**Issue by jsms90** [foundersandcoders/master-reference#260]
## Why
+ Our current plan is London-centric & current cohort-centric
+ [Why that needs to change](#why-that-needs-to-change)

## What
+ Invite the wider community into curriculum planning
+ Develop the `master-reference` together - no forks

## How
**Process**
+ Bring proposals & the minutes from curriculum planning into `master-reference` (non-campus specific) DONE
+ Someone runs a similar day for Nazareth. After brainstorming, FACN1 looks through the existing list of proposals, then comments and/or raises their own. Whoever presents these to students should _heavily_ encourage participation, so that Nazareth are able to get involved in this conversation
+ Send out invites ASAP!
+ Create a contributing guide for community-wide participation

**Content**
+ Each week is a "component" of the master reference.
  + Weeks in `master-reference` should be renamed to refer to their content, not their position in the course. New weeks go in new folders in the `master-reference/coursebook`.
  + Each campus creates their own [`timeline.md`](https://github.com/foundersandcoders/master-reference/blob/master/coursebook/README.md) which references the components they have chosen
  + The inclusion of existing weeks or creation of new ones is up to each campus.
+ Large scale changes to the curriculum (e.g. removal of research afternoon) are made through proposals. These need to be agreed by the community, so the onus is on the proposal-raiser to provide evidence e.g. s-g-cs or old issues.
+ Iterations on an existing component are done collectively.
+ Each campus is empowered to create new components. Running this new component should be an attempt to confirm / reject a defined hypothesis about the improvement that it would make.

# TL;DR

## Why that needs to change
+ Forking the `master-reference`
  + You cannot raise issues on forks
  + Would you fork every workshop too?
  + Forking implies that there is an expectation of it getting merged down the line. What if an "experiment" works in one location, but not in another. It will forever be relegated to a forked version.
  + Creating multiple versions of a week, each with slight differences, would make global maintenance impossible.
+ Nazareth should be as involved in curriculum planning as possible. Especially in decisions that effect the design of the whole course
+ Curriculum design should be influenced by other members of FAC than just the current cohort
  + In the history of FAC, leaving this up to the current cohort alone has caused a flip-flop effect, instead of continuous improvement
  + In the history of FAC, the current cohort has sometimes been told they have autonomy over the content, and then had their decision overturned (Hapi vs Express). Let's avoid this happening again by properly involving the wider community during the decision-making process
+ The current process discourages involvement from the wider community
  + this is perceived to be the domain of the current cohort alone, so alumni feel like they are imposingif they speak their minds & influence the opinion of the room
  + the wider community hasn't been notified in advance & daytime on a Tuesday is not accessible for many other than the current cohort, so we are unlikely to hear from the wisdom and experience of the rest of our community

**Long term future**
+ Next time we do this, let the wider community know that this day will be happening well in advance
+ Provide a stronger contributing guide for the `master-reference`, to enable participation to the master reference #248 
+ Once people are used to `master-reference` as an open source project & know _how_ they can contribute effectively, only those who contribute to the curriculum should be given votes at the meeting

**Comment by jsms90** [comment]
Discussion about how to run curriculum planning doesn't really belong in our next meeting, and this is also more of a brain dump than a proper proposal :sweat_smile:

Going to rethink this, and we'll probably document the process after this iteration anyway :+1:
